src/WCSimWCAddDarkNoise.cc

```diff
@@ -323,6 +324,7 @@ void WCSimWCAddDarkNoise::AddDarkNoiseBeforeDigi(WCSimWCDigitsCollection* WCHCPM
 	  (*WCHCPMT)[ list[noise_pmt]-1 ]->SetPhotonEndPos(PMTindex[noise_pmt],(*WCHCPMT)[ list[noise_pmt]-1 ]->GetPos());
 	  (*WCHCPMT)[ list[noise_pmt]-1 ]->SetPhotonStartDir(PMTindex[noise_pmt],-(*WCHCPMT)[ list[noise_pmt]-1 ]->GetOrientation());
 	  (*WCHCPMT)[ list[noise_pmt]-1 ]->SetPhotonEndDir(PMTindex[noise_pmt],-(*WCHCPMT)[ list[noise_pmt]-1 ]->GetOrientation());
+    (*WCHCPMT)[ list[noise_pmt]-1 ]->SetPhotonCreatorProcess(PMTindex[noise_pmt],kUnknownProcess);
```

Comment:
```suggestion
    (*WCHCPMT)[ list[noise_pmt]-1 ]->SetPhotonCreatorProcess(PMTindex[noise_pmt],kDarkNoise);
```

Reply:
Same as above, kUnknownProcess is not only used for dark noisy but also as the creatorProcess of the primaries.

Reply:
👍